Project.toml

```diff
@@ -2,21 +2,24 @@ name = "GasPowerModels"
 uuid = "1fcaf345-95e3-5b8b-91d6-2145eb844b9d"
 authors = ["Russell Bent"]
 repo = "https://github.com/lanl-ansi/GasPowerModels.jl"
-version = "0.1.1"
+version = "0.2.0"
 
 [deps]
 GasModels = "5e113713-6c35-5477-b766-e1109486666f"
 InfrastructureModels = "2030c09a-7f63-5d83-885d-db604e0e9cc0"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 PowerModels = "c36e90e8-916a-50a6-bd94-075b64ef4655"
 
 [compat]
-GasModels = "~0.4"
-InfrastructureModels = "~0.2"
-JSON = ">= 0.18"
-PowerModels = "~0.12"
+GasModels = "~0.7"
+InfrastructureModels = "~0.5"
+JSON = "~0.18, ~0.19, ~0.20, ~0.21"
+JuMP = "~0.20, ~0.21"
+MathOptInterface = "~0.8, ~0.9"
+PowerModels = "~0.17"
```

Comment:
You will need to add a `Memento` version in here to pass the automerge check..

Reply:
Also might check if you can drop `MathOptInterface` from the deps here.  Maybe it is only needed for tests?

Reply:
All right, these have both been addressed. `MathOptInterface` is dropped completely, and exported statuses are grabbed from `IM._MOI`.